boards/tdk/robokit1/robokit1_defconfig

```diff
@@ -9,3 +9,7 @@ CONFIG_WDT_DISABLE_AT_BOOT=y
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 CONFIG_SERIAL=y
+
+CONFIG_I2C_RTIO=y
+CONFIG_SPI_RTIO=y
+CONFIG_SPI_SAM_DMA=y
```

Comment:
Need to do this in `Kconfig.robokit1` to avoid overriding the dependencies on CONFIG_I2C and CONFIG_SPI.